src/modules/sensors/CMakeLists.txt

```diff
@@ -35,9 +35,8 @@ px4_add_module(
 	MODULE modules__sensors
 	MAIN sensors
 	PRIORITY "SCHED_PRIORITY_MAX-5"
-	STACK_MAIN 1200
+	STACK_MAIN 2000
```

Comment:
That's a steep increase - does the start handler really need more stack? Or did you intent to increase the main app stack? Which would be in task_spawn, NOT here.